File: Famoser.MassPass.Data/Enum/RemoteStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Famoser.MassPass.Data.Enum
{
    /// <summary>
    /// response from the server; indicates if contentmodel is up to date or not
    /// </summary>
    public enum RemoteStatus
    {
        UpToDate = 1,
        Changed = 2
    }
}

File: Famoser.MassPass.Api/src/Types/RemoteStatus.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 29/05/2016
 * Time: 19:28
 */

namespace Famoser\MassPass\Types;


class RemoteStatus
{
//[Description("up to date")]
    const UpToDate = 1;
//[Description("changed")]
    const Changed = 2;
//[Description("not found on server")]
    const NotFound = 3;
}